gradle.properties

```diff
@@ -19,6 +19,7 @@ micronautDataVersion=2.3.1
 micronautSpringVersion=2.1.1
 eventuateUtilVersion=0.14.0.BUILD-SNAPSHOT
 r2dbcMysqlVersion=0.8.2.RELEASE
+r2dbcPostgresVersion=0.9.0.M2
```

Comment:
should we use 0.8.8.RELEASE instead ? 0.9.0.M2 is using R2DBC SPI 0.9.0.M2
and r2dbcMysqlVersion=0.8.2.RELEASE is still using R2DBC SPI 0.8.x version